traffic_portal/app/src/traffic_portal_properties.json

```diff
@@ -407,6 +407,14 @@
           "url": "http://trafficcontrol.apache.org/"
         }
       ]
+    },
+    "oAuth": {
+      "_comment": "Opt-in OAuth properties for SSO login.  oAuthUrl is the url to redirect to for your oAuth provider.  oAuthTokenQueryParam is the query parameter key that the oAuth provider will use for the token (defaults to access_token).  redirectUriParameterOverride is to override if the oAuth provider requires a different key for the redirect_uri parameter (defaults to redirect_uri if empty). clientId is the client to be sent to the oAuth provider in the client_id field.",
```

Comment:
Now that i think about it more, maybe it makes more sense to just have a link to http://traffic-control-cdn.readthedocs.io/en/latest/admin/quick_howto/oauth_login.html in here. what do you think?

Reply:
i like that! ill update it. i might keep the defaults in there too so that its a quick and easy thing to reference

Reply:
In general linking to the latest docs is dangerous - someone who deploys 4.0 can get directed to the 5.0 docs.

Reply:
good call.  done